Features/huidig.feature

```diff
@@ -31,4 +32,46 @@ Dan bevat de response de verblijfsobjecten met een eindstatus niet en de verblij
 #### Scenario:
 Gegeven een pand met een eindstatus
 Wanneer op het endpoint verblijfsobjecten de verblijfsobjecten in dat pand worden opgevraagd met de parameter 'huidig'
-Dan volgt een 404 foutmelding dat het pand niet gevonden kan worden.
+Dan worden er geen verblijfsobjecten geretourneerd
+
+#### Scenario: het adressen endpoint levert altijd huidige objecten
+Gegeven het adressen endpoint retourneert alleen huidige objecten
+En er bestaat een nummeraanduiding met status 'Naamgeving ingetrokken'
+Wanneer het adressen endpoint voor deze nummeraanduiding wordt bevraagd
+Dan wordt het adres niet geretourneerd
+
+#### Scenario: het adressen endpoint levert altijd huidige objecten en retourneert een object niet als een gerelateerd object een eindstatus heeft
+Gegeven het adressen endpoint retourneert alleen huidige objecten
+En er bestaat een nummeraanduiding met status 'Naamgeving uitgegeven'
+En het gerelateerde verblijfsobject heeft status 'Verblijfsobject ingetrokken'
+Wanneer het adressen endpoint voor deze nummeraanduiding wordt bevraagd
+Dan wordt het adres niet geretourneerd
+
+#### Scenario: het adressen endpoint levert altijd huidige objecten
+Gegeven het adressen endpoint retourneert alleen huidige objecten
+En er bestaat een nummeraanduiding met status 'Naamgeving uitgegeven'
+En het gerelateerde verblijfsobject heeft status 'Verblijfsobject in gebruik'
+Wanneer het adressen endpoint voor deze nummeraanduiding wordt bevraagd
+Dan wordt het adres geretourneerd
+
+#### Scenario: het adressen endpoint levert altijd huidige objecten tenzij op basis van een identificatie van een pand wordt bevraagd, het pand kan een eindstatus hebben en het adres wordt dan wel geretourneerd

```

Comment:
Scenario: het adressen uitgebreid endpoint levert WEL adressen wanneer deze gerelateerd is aan een pand met een eindstatus.